appdev/index.rst

```diff
@@ -25,6 +25,13 @@ Tools
 
 `Clickable <http://clickable.bhdouglass.com/en/latest/>`_ is a meta-build system for Ubuntu Touch applications that allows you to compile, build, test and publish ``click`` packages and provides various templates to get you started with app development. It is currently the easiest and most convenient way of building click packages for Ubuntu Touch. You can use any code editor or IDE that you choose and build your apps from the commandline with Clickable.
 
+Code Editor Integrations
+------------------------
+
+Use clickable with the `Atom Editor <https://atom.io>`__ by installing `atom-clickable-plugin <https://atom.io/packages/atom-clickable-plugin>`__.
+This is an fork of the original (now unmaintained) `atom-build-clickable <https://atom.io/packages/atom-build-clickable>`__
```

Comment:
Not a native speaker, but I'm pretty sure it should be "a fork"

However, I wonder whether we really need that sentence at all. It's not too relevant for people looking to make UT apps that this plugin was fork from some other, now unmaintained version...

Reply:
Whoops, yes of course that should be "a fork". I must have read that sentence loads of times and never realised :slightly_smiling_face: 

I agree with your second point though, that sentence does more harm than good. I will remove it.